### ISSUE TITLE
footer designs [MOBILE] 

### ISSUE THREAD

**Issue by tanmoy27112000** [tanmoy27112000/flutter_web_base#8]
design link: https://getbootstrap.com/docs/5.3/examples/footers/

**Comment by ayesha-2101** [comment]
@tanmoy27112000 can you elaborate a bit ?

**Comment by tanmoy27112000** [comment]
@ayesha-2101 please follow up on this in JWOC discord channel if you have any issue

**Comment by ayesha-2101** [comment]
what do I do in this @tanmoy27112000 ? do I have to design the frontend of the footer

**Comment by tanmoy27112000** [comment]
@ayesha-2101 yes you need to replicate the design from the bootstrap footer design in flutter

**Comment by sneha502** [comment]
Please assign me this issue under JWOC

**Comment by tanmoy27112000** [comment]
@sneha502 sure
